en/docs/administer/logging-and-monitoring/logging/gw-api.yaml

```diff
@@ -0,0 +1,167 @@
+swagger: '2.0'
+info:
+  title: WSO2 API Manager - Gateway
+  version: v0.9
+  description: This document specifies a **RESTful API** for WSO2 **API Manager** - Gateway
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/api-manager/'
+    email: architecture@wso2.com
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+basePath: /api/am/gwapi/v0.9
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /api-logging:
+    post:
+      summary: |
+        POST logging data of the API with its context template and version
+      description: |
+        This operation enables you to provide the API context template(context/version) with the log level (headers, body or all). You should either provide a payload or context and logLevel as query parameters with this API request.
+      parameters:
+        - name: Payload
+          description: |
+            context and the logLevel can be provided as a payload. This is useful if you want to enable logs for multiple APIs at once.
+          required: false
+          in: body
+          schema:
+            $ref: '#/definitions/APIList'
+        - name: context
+          in: query
+          description: |
+            API context ( context/version eg: pizzashack/1.0.0)
```

Comment:
if you give context/version of the API, how do you enable it for multiple APIs? shall we add an example?

Reply:
This is already shown once you expand the swagger console.Reference scheme has been provided.